### ISSUE TITLE
Graphics card requirements for training

### ISSUE THREAD

**Issue by ouyangziyao** [happinesslz/LION#6]
Hello, this is an excellent work. Can I retrain this model on a graphics card with only 12GB of VRAM?

**Comment by happinesslz** [comment]
@ouyangziyao Thanks for you attention! Yes, you could have a try. You could reduce the batchsize of 2~(24G for batchsize=2) to 1 or reduce the channel dimension of 128 to 64 or even 32. Note that if reducing the batchsize, the corresponding learning rate need to be adjusted.

**Comment by ouyangziyao** [comment]
Thank you for your valuable feedback.

**Comment by xinpuliu** [comment]
@happinesslz 你好，如果单卡的batch=1了，学习率要怎么调整呀？
